cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Cardano's minimum viable governance model as described within [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md) introduces the concept of a Cardano constitution.
+Although CIP-1694 gives no definition to the constitution's content or form.
+
+This proposal aims to describe a standardized form for the Cardano constitution.
+Aiming to enhance the accessibility, readability and safety of the document.
+
+> **Note:** This proposal only covers the technical form of the constitution, this standard is agnostic to the content of the constitution.
+
+## Motivation: why is this CIP necessary?
+
+CIP-1694 defines the on-chain anchor mechanism used to link the off-chain constitution document to on-chain actions.
+This mechanism was chosen due to its simplicity and cost effectiveness, moving the potentially large Cardano constitution off chain, referencing it via a hash digest and URI.
+This is the extent to which CIP-1694 outlines the constitution, it does provide suggestions around hashing algorithm, off-chain storage location, rich text styling.
+
+By formalizing the form of the constitution and it's iterations, we aim to promote it's longevity and accessibility.
+This is essential for the governance model
+
+Like the Cardano constitution this standard impacts every ada holder.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution, we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, we can ensure it's immutability and permissionless access.
+This is a step to ensuing the longevity and accessibility of each constitution iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make roles constitution iteration comparison tools easy, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification should be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### File Type
+// todo - any suggestions?
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text must be in its raw un-rendered format.
+
+- add a way to change algorithm?
```

Comment:
> the algorithm is specified in the constitution update governance action

This would make it tricky to be able to discover the current hash algorithm